chat-client/src/contracts/serverContracts.ts

```diff
@@ -54,6 +54,7 @@ export type ServerMessageCommand =
 
 export interface Message {
```

Comment:
nit: it's not clear why we have `Message` as a separate interface - you could merge it with `ServerMessage`.